ext/datadog_profiling_native_extension/stack_recorder.c

```diff
@@ -193,6 +193,10 @@ struct stack_recorder_state {
   pthread_mutex_t mutex_slot_two;
   profile_slot profile_slot_two;
 
+  ddog_prof_ManagedStringStorage string_storage;
+  uint32_t label_key_allocation_class;
+  uint32_t label_key_gc_gen_age;
```

Comment:
Minor: It occurs to me that it may be nice for libdatadog to specify a typedef (of a one-item-struct) to represent this, so we get a better type than "a naked integer".